### DISCUSSION TITLE
add `neil try dependency` like lein-try

### DISCUSSION THREAD

**Discussion by brunoti** [babashka/neil#197] [Ideas]
I think for people who are trying to learn the Clojure CLI, neil is a very good helper. Also, one thing that can help people move faster is [lein-try](https://github.com/avescodes/lein-try): a helper for trying dependencies with no overhead. 

A neil try would be very cool to have. I don't think I have enough expertise to add it myself and create a PR unless I have a lot of help. Still, a nice feature.

**Comment by borkdude** [comment]
There's also a project like this called: https://github.com/eval/deps-try
